### ISSUE TITLE
Apostrophe and accent letters cannot be used in Live Search queries

### ISSUE THREAD

**Issue by lagunacellar** [magento/magento2#35958]
### Preconditions and environment

- Magento 2.4.4 & 2.4.5
- Magento Cloud, Live Search

### Steps to reproduce

1. Search for products using queries that contain apostrophe or accent letters.

<img width="528" alt="search" src="https://user-images.githubusercontent.com/64744041/184948864-a73b3767-c8f8-4f2f-9380-3aa5ec70df34.png">

### Expected result

Expected products matching the search terms to be returned in search result.

### Actual result

No product is found if the product name contains apostrophe or accent letters.

### Additional information

Live Search cannot handle any queries that contain apostrophe or accent letters. These accent letters are frequently used in product names of French and/or other European origin.

See screenshots below, the query of L'If returns no results, yet multiple versions of the product exist and are visible when queried by SKU.  Identical failure to return Live Search results occurs when queries include accented letters such as â, î, é, è ...

<img width="598" alt="search by sku" src="https://user-images.githubusercontent.com/64744041/184948984-a20bd496-8088-4a64-86e7-cc0cde8edb68.png">

### Triage and priority

- [X] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by lagunacellar** [comment]
@magento give me 2.4-develop instance - upcoming 2.4.x release

**Comment by lagunacellar** [comment]
This issue is strictly limited to Live Search service on Magento Cloud.  If using the plain vanilla ElasticSearch, the search results appear as expected.

**Comment by engcom-Dash** [comment]
@magento give me 2.4-develop instance

**Comment by engcom-Dash** [comment]
Hi @lagunacellar ,

We are trying to reproduce the issue.But we are not able to reproduce in Both 2.4.4 and 2.4.5 and we are Getting expected results.

Here i attached the screenshots for your reference:

![Screenshot from 2022-08-17 15-38-06](https://user-images.githubusercontent.com/60198592/185093543-b77c9d3a-0c0e-4ae9-8a04-84eada5e4d83.png)
![Screenshot from 2022-08-17 15-34-44](https://user-images.githubusercontent.com/60198592/185093626-d69ae448-3bc2-46ea-9d55-c61db1070096.png)


Steps to reproduce:
1. Install 2.4..4 and 2..4.5 Instance
2. Create the Product with Accent Letters
3. Search product with respective Product name
4. Product will be find in Search results

Let us know if we missing anything

Thanks!!!

**Comment by lagunacellar** [comment]
@engcom-Dash , thank you for looking into this.  Indeed, this problem only happens on Live Search, the premium Magento Commerce feature.  It would appear that Live Search, using Adobe data services, blocks accent letters and special characters as inputs.  When not using Live Search, Magento 2.4 has no problem handling accent letter, apostrophe, etc, even if it's Magento Cloud hosted.

I tried first reporting the issue on https://github.com/magento/magento-cloud, but was asked to report it to https://github.com/magento/magento2, even though this issue is limited to Magento Cloud using Live Search.  I have opened a ticket with Magento Support.

If this is not something the community can work on, as it seems to be related to Adobe's data services, we can close this and leave it up to Adobe dev team and tech support to sort it out.
